.travis.yml

```diff
@@ -4,9 +4,11 @@ jdk:
   - oraclejdk8
 env:
   matrix:
-    - SCRIPT="sbt -jvm-opts .travis-jvmopts test"
-    - SCRIPT="sbt -jvm-opts .travis-jvmopts publishLocal scripted"
-    - SCRIPT="sbt -jvm-opts .travis-jvmopts unidoc && cd docs && sbt -jvm-opts .travis-jvmopts markdownValidateDocs test markdownEvaluateSbtFiles"
+    - SCRIPT="sbt --warn -jvm-opts .travis-jvmopts test"
+    # Run scripted in a separate process without --warn so we can see its progress
```

Comment:
Look carefully, sbt is invoked twice (separated by &&, this gets evaled by bash so && is a delimiter for multiple commands, that will only execute the second command if the first was successful).  The first one is invoked with --warn, sbt goes very quiet when you do this, in fact it can be a problem because travis will time the build out if it doesn't log anything.  The second invocation of sbt, the one that executes scripted, doesn't use --warn.